GameServer/packets/Server/PacketLib1124.cs

```diff
@@ -3134,80 +3133,34 @@ public virtual void SendRealm(eRealm realm)
 				SendTCP(pak);
 			}
 		}
-		public virtual void SendRegions()
+		public virtual void SendRegions(ushort regionId)
```

Comment:
Seeing that we are only sending one region now, should we make this singular?
`SendRegion`?

Reply:
I stay conservative about changes on public methods to keep a better compatibility with existent scripts or custom code.